modules/apps/commerce/commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/configuration/CPSortPortletInstanceConfiguration.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.commerce.product.content.search.web.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Balazs Breier
+ */
+@ExtendedObjectClassDefinition(
+	category = "catalog",
+	scope = ExtendedObjectClassDefinition.Scope.PORTLET_INSTANCE
+)
+@Meta.OCD(
+	id = "com.liferay.commerce.product.content.search.web.internal.configuration.CPSortPortletInstanceConfiguration",
+	localization = "content/Language",
+	name = "commerce-product-sort-portlet-instance-configuration-name"
+)
+public interface CPSortPortletInstanceConfiguration {
+
+	@Meta.AD(name = "select-default-sort", required = false)
+	public String selectDefaultSort();
```

Comment:
can you please check if we have other configuration with "a select" and how are they called? I don't like too much the select*** name

Reply:
I cannot find any select inside *PortletInstanceConfiguration.java classes so call it only default-sort

please verify all the other points you're using this name in the PR